can-bus/json/DIM/DIM_alerts.json

```
@@ -1,6 +1,17 @@
 {
-    "msg_id": 500,
-    "alerts": [
-        "DIM_ALERT_STARTUP"
+    "warnings_id": 500,
+    "faults_id": 501,
+    "warnings": [
+        "DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_1HZ",
+        "DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_100HZ",
+        "DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_1KHZ",
+        "DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_CANRX",
+        "DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK_CANTX",
+        "DIM_WARNING_WATCHDOG_TIMEOUT",
+        "DIM_WARNING_TX_OVERFLOW",
+        "DIM_WARNING_RX_OVERFLOW"
+    ],
+    "faults": [
+        "DIM_FAULT_MISSING_HEARTBEAT"
     ]
 }
```

nit: missing blank line
```suggestion
}

```